### PR TITLE
feat(central): only fetch changelog on user request

### PR DESCRIPTION
**Pull request by connoratrug**
What are the main changes you did:
- Not all admin / manager user want to see the changelog ( last updated schema ) all the time as it takes some time to query this for all the schema's. This pr places the data request behind a button 

how to test:
- Not admin or manger , nothing changes , no btn is shown
- as admin or manger by default no change log data is shown ( fetched ) , but a btn is shown to request the data
- if the btn is clicked the data is fetched 

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

### CODE REVIEW

**Comment by YpeZ** [comment]
I like it. Works very smoothly. Couple of comments

- I added an entry in a table in each schema. When clicking the button, I can see that queries are launched, but the cells are not populated
- The spacing looks a bit awkward at the moment, but I suppose that's low priority

File: apps/central/src/components/Groups.vue [hunk 1]
```diff
@@ -18,7 +18,21 @@
...
+        v-if="showChangeColumnButton && !showChangeColumn"
+        @click="showChangeColumn = true"
+        class="float-right"
+        >Show Changelog</ButtonOutline
+      >
+      <ButtonOutline
+        v-if="showChangeColumnButton && showChangeColumn"
+        @click="showChangeColumn = false"
+        class="float-right"
+        >Hide Changelog</ButtonOutline
```
Comment (YpeZ) [inline apps/central/src/components/Groups.vue:33]: Change to `Hide changelog`, no capital `C`, same in line 27

File: apps/central/src/components/Groups.vue [hunk 2]
```diff
@@ -18,7 +18,21 @@
...
+        @click="showChangeColumn = true"
+        class="float-right"
+        >Show changelog</ButtonOutline
+      >
+      <ButtonOutline
+        v-if="showChangeColumnButton && showChangeColumn"
+        @click="showChangeColumn = false"
+        class="float-right"
+        >Hide changelog</ButtonOutline
+      >
```
Comment (chinook25) [inline apps/central/src/components/Groups.vue:34]: Layout of the <> brackets is ugly, could you change it to:
<tag>
  content
</tag>
?

File: apps/central/src/components/Groups.vue [hunk 3]
```diff
@@ -282,6 +299,13 @@ export default {
       }
     },
   },
+  watch: {
+    showChangeColumn: function (val) {
```
Comment (chinook25) [inline apps/central/src/components/Groups.vue:303]: usually we use the ()=>{} style function in the watches, why this time function(){} ?

File: apps/central/src/components/Groups.vue [hunk 4]
```diff
@@ -204,7 +221,7 @@ export default {
         .then((data) => {
           this.schemas = data._schemas;
           this.loading = false;
-          if (this.hasManagerPermission) {
+          if (this.hasManagerPermission && this.showChangeColumn) {
             this.fetchChangelogStatus();
```
Comment (chinook25) [inline apps/central/src/components/Groups.vue:225]: if showChangeColumn is initially false and is changed to true later, doesn't the fetchChangeLogStatus also need to happen? Does this if need to be moved to the watch?

**Review by chinook25 — CHANGES_REQUESTED** [review]

**Review by chinook25 — APPROVED** [review]

**Pull request merged**